DEVELOPMENT.md

```diff
@@ -49,6 +49,12 @@ To package apps for the local platform:
 npm run package
 ```
 
+:bulb: When packaging for MacOS make sure to set your signing/notirization env variables per [the electron-builder docs](https://www.electron.build/mac#notarize):
+
+```bash
+APPLE_ID='YOUR_APPLE_ID' APPLE_APP_SPECIFIC_PASSWORD='YOUR_APP_PASSWORD' APPLE_TEAM_ID='YOUR_TEAM_ID' npm run package
```

Comment:
They recommend using APPLE_API_KEY, APPLE_API_KEY_ID and APPLE_API_ISSUER. We should update the docs to use that instead 🙏

Reply:
Notarizing was also not working until I added `notarize: true` to the `mac` block in `package.json`.

Reply:
I'm running an arm mac and this makes a pure arm64 build, which means this wouldn't work on older intel macs. I think we need to build a universal dmg.

Reply:
If you replace

```
      "target": [
        "dmg"
      ],
```

with

```
      "target": [
        {
          "target": "dmg",
          "arch": [
            "universal"
          ]
        }
      ],
```

then it builds a universal dmg.

Reply:
Also, can you add a note for something like

```
You can obtain an API key from [App Store Connect](https://appstoreconnect.apple.com/access/integrations/api).
Create a **Team Key** (not an _Individual Key_) with **App Manager** access.
```